lib/index.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
+    <title><%= htmlWebpackPlugin.options.title %></title>
+    <meta name="app-build-reference" content="<%= htmlWebpackPlugin.options[ 'build-reference' ] %>">
+    <meta name="node-version" content="<%= htmlWebpackPlugin.options[ 'node-version' ] %>">
+    <meta name="build-platform" content="<%= htmlWebpackPlugin.options[ 'build-platform' ] %>">
```

Comment:
@beaucollins here I have added what I think are three important indicators of the build generator based on your comments in this PR. Please feel free to change them and add or take them away.